### ISSUE TITLE
Delay parameter of `Tween.interpolate_property` seems to be ignored in tween sequences

### ISSUE THREAD

**Issue by kubecz3k** [godotengine/godot#6864]
**Operating system or device - Godot version:**
This issue applies to https://github.com/godotengine/godot/pull/6613 commit

**Issue description** (what happened, and what was expected):
Let's say I have this simple function to  move object to some target position, and then return to it's previous position. Let's say I'm doing this with tween by starting two tweens (one of them is starting with delay)
here is the code:

```
func tweenToAndReturn(inTargetPos):
    var modelCurrentPos = object.get_translation();
    tween.interpolate_property(object, "transform/translation", modelCurrentPos, inTargetPos,0.3, Tween.TRANS_QUAD, Tween.EASE_IN);
    tween.interpolate_property(object, "transform/translation", inTargetPos, modelCurrentPos,0.3, Tween.TRANS_QUAD, Tween.EASE_OUT,0.3);
    tween.start();
```

Here is how it works before #6613  (or with #6613 but also with  #6863) 
![tween-sequence-1](https://cloud.githubusercontent.com/assets/6129594/19494768/e112192c-9580-11e6-8440-58daab39f53d.gif)

Here is how it works with #6613
![tween-sequence-2](https://cloud.githubusercontent.com/assets/6129594/19494799/05cdd7ba-9581-11e6-8007-21a09722184a.gif)

I'm not sure what's happening exactly, but first thought is that delay parameter of `interpolate_property` is for some reason ignored in #6613 (or this interpolation is already killed at the time it need to start)

**Steps to reproduce:**
1. Check sample project without  #6613 testCube will be able to return to it's initial position
2. Apply #6613 testCube won't be able to return to it's initial position

**Link to minimal example project** (optional but very welcome):
[TweenSequenceAndDelay.zip](https://github.com/godotengine/godot/files/537269/TweenSequenceAndDelay.zip)

**Comment by akien-mga** [comment]
Ping @pkowal1982.

**Comment by pkowal1982** [comment]
Old implementation of `remove()` deletes only first interpolation matching.
If there are next ones it won't remove them and I think it's not good behaviour, even if we document it.

When interpolation becomes inactive it calls internally the same `remove()` and in new implementation causes to remove also delayed interpolations.

I think best we can do is to remove all matching interpolations on explicit call and internally remove only the first one which became inactive. We'll have consistent api on tween and working interpolation stacking.

**Comment by Barina** [comment]
A workaround would be:
```
var t = get_node("Tween")
var s = get_node("Sprite")
t.interpolate_method(s, "set_scale", Vector2(1, 1), Vector2(), 1, t.TRANS_LINEAR, t.EASE_IN)
t.interpolate_method(s, "set_scale", Vector2(), Vector2(1, 1), 1, t.TRANS_LINEAR, t.EASE_IN, 1)
t.interpolate_callback(t, 2, "set_repeat", false)
t.set_repeat(true)
t.start()
```

**Comment by bojidar-bg** [comment]
What happened with this issue?

**Comment by pkowal1982** [comment]
Should be fixed with #6918
Still there's a problem when you add delayed tween before non delayed one. Probably there should be a flag marking a tween for deletion which would solve this.

**Comment by akien-mga** [comment]
Closing as fixed then.

> Still there's a problem when you add delayed tween before non delayed one. Probably there should be a flag marking a tween for deletion which would solve this.

Maybe open a new issue for that.
